base_external_dbsource_mysql/models/base_external_dbsource.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2011 Daniel Reis
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from odoo.addons.base_external_dbsource.models import base_external_dbsource
+
+    CONNECTORS = base_external_dbsource.BaseExternalDbsource.CONNECTORS
+    try:
+        import MySQLdb
+
+        CONNECTORS.append(("mysql", "MySQL"))
+        assert MySQLdb
+    except (ImportError, AssertionError):
+        _logger.info(
+            'MySQLdb not available. Please install "mysqlclient" python package.'
+        )
+    try:
+        import sqlalchemy
+    except ImportError:
+        _logger.info(
+            "SQLAlchemy library not available. Please "
+            'install "sqlalchemy" python package.'
+        )
+except ImportError:
+    _logger.info("base_external_dbsource Odoo module not found.")
```

Comment:
This can not happen, as you have it in the depends of this module. If module you depend on is not there, your module will not load and this line will never be reached.